src/main/webapp/i18n/de/organizationManagement.json

```diff
@@ -18,19 +18,19 @@
             "title": "Organisation"
         },
         "userSearch": {
-            "usersPerPage": "{{number}} Benutzer pro Seite",
-            "showAllUsers": "Alle Benutzer anzeigen",
-            "searchForUsers": "Benutzer zu dieser Organisation hinzufügen durch Suchen nach Login oder Namen",
-            "searchNoResults": "Keine Benutzer gefunden",
+            "usersPerPage": "{{number}} User pro Seite",
+            "showAllUsers": "Alle User anzeigen",
+            "searchForUsers": "User zu dieser Organisation hinzufügen durch Suchen nach Login oder Namen",
+            "searchNoResults": "Keine User gefunden",
             "removeFromOrganization": {
-                "modalQuestion": "Soll der Nutzer mit dem Login <strong>{{ title }}</strong> wirklich aus dieser Organisation entfernt werden?"
+                "modalQuestion": "Soll der User mit dem Login <strong>{{ title }}</strong> wirklich aus dieser Organisation entfernt werden?"
             }
         },
         "modalSelector": {
             "select": "Auswählen",
             "title": "Organisationen"
         },
-        "patternChangeWarning": "Beim ändern des Emailmusters, wird eine Neuzuweisung der Nutzern zu dieser Organisation ausgeführt"
+        "patternChangeWarning": "Beim Ändern des Email Musters, wird eine Neuzuweisung der User zu dieser Organisation ausgeführt"
```

Comment:
```suggestion
        "patternChangeWarning": "Beim Ändern des Email-Musters, wird eine Neuzuweisung der User zu dieser Organisation ausgeführt"
```

Reply:
I changed it in the file to Email-Muster, to be consistent